packages/common/ide/.eslintrc.json

```diff
@@ -5,7 +5,7 @@
       {
         "patterns": [
           {
-            "group": ["../../*"],
+            "group": ["@cursorless/*", "../../*"],
```

Comment:
Hm, don't nested eslintrcs cascade? This looks identical to the parent one.

Is it explicitly overriding this rule for some reason?

Reply:
The relative exclusions (`../../`) need to be different at each directory level

Note that all of these eslint files will be going away in the next few days when we migrate to pnpm. Just added these to help me spot issues before migration